src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -322,4 +397,37 @@ private function createOrder(
 
         return $order;
     }
+
+    /**
+     * @param $count
+     *
+     * @return CustomerInterface[]
+     */
+    private function generateCustomers($count)
+    {
+        $customers = [];
+
+        for ($i = 0; $i < $count; $i++) {
+            $customer = $this->customerFactory->createNew();
+            $customer->setEmail(sprintf('john%s@doe.com', uniqid()));
+            $customer->setFirstname('John');
+            $customer->setLastname('Doe'.$i);
+
+            $customers[] = $customer;
+
+            $this->customerRepository->add($customer);
+        }
+
+        return $customers;
+    }
+
+    /**
+     * @param string $price
+     *
+     * @return int
+     */
+    private function getPriceFromString($price)
```

Comment:
Do we really need this method? It is context, so why can't we use a lexical context?